src/main/java/duke/Parser.java

```diff
@@ -0,0 +1,101 @@
+package duke;
+
+import duke.tasks.*;
+
+import java.time.LocalDate;
+import java.time.LocalDateTime;
+
+public class Parser {
+    /**
+     * Parses user input to a ToDo object
+     * @param input the user input given
+     * @return a ToDo object
+     */
+    public static ToDo parseToDo(String input) {
+        String td = input.substring(5);
+        try {
+            if (td.equals("")) {
+                throw new DukeExceptions("Description cannot be empty!");
+            }
+        } catch (DukeExceptions e) {
+            System.out.println("Description cannot be empty!");
```

Comment:
I think using e.getMessage() here to get the Duke exception message written above instead of repeating it twice might be a better option.